test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/task/IncrementalTask.java

```diff
@@ -0,0 +1,117 @@
+    
+    private final DataSource dataSource;
+    
+    private final String orderTableName;
+    
+    private final String insertTableSql;
+    
+    private final KeyGenerateAlgorithm primaryKeyGenerateAlgorithm;
+    
+    private final DatabaseType databaseType;
+    
+    private final int loopCount;
+    
+    @Override
+    public void run() {
+        List<Object[]> orderInsertData = PipelineCaseHelper.generateOrderInsertData(databaseType, primaryKeyGenerateAlgorithm, loopCount);
+        List<Object> primaryKeys = new LinkedList<>();
+        for (Object[] each : orderInsertData) {
+            primaryKeys.add(each[0]);
+            DataSourceExecuteUtil.execute(dataSource, insertTableSql, each);
+        }
+        ThreadLocalRandom random = ThreadLocalRandom.current();
+        for (int i = 0; i < Math.max(1, loopCount / 4); i++) {
+            // TODO 0000-00-00 00:00:00 now will cause consistency check failed of MySQL.
+            // DataSourceUtil.execute(dataSource, String.format("UPDATE %s SET t_datetime='0000-00-00 00:00:00' WHERE order_id = ?", orderTableName)
+            updateOrderById(primaryKeys.get(random.nextInt(0, primaryKeys.size())));
+            if (databaseType instanceof MySQLDatabaseType) {
+                setNullToUnsignedFields(primaryKeys.get(random.nextInt(0, primaryKeys.size())));
+            }
+        }
+        for (int i = 0; i < Math.max(1, loopCount / 5); i++) {
+            deleteOrderById(primaryKeys.get(random.nextInt(0, primaryKeys.size())));
+        }
+        log.info("increment task runnable execute successfully.");
+    }
+    
+    private void updateOrderById(final Object orderId) {
+        int randomInt = ThreadLocalRandom.current().nextInt(-100, 100);
+        if (databaseType instanceof MySQLDatabaseType) {
+            String updateSql = String.format(MYSQL_UPDATE_ORDER_BY_ID, orderTableName);
+            int randomUnsignedInt = ThreadLocalRandom.current().nextInt(10, 100);
+            LocalDateTime now = LocalDateTime.now();
+            DataSourceExecuteUtil.execute(dataSource, updateSql, new Object[]{"中文测试", randomInt, randomInt, randomInt, randomUnsignedInt, randomUnsignedInt, randomUnsignedInt,
+                    randomUnsignedInt, 1.0F, 1.0, new BigDecimal("999"), now, now, now.toLocalDate(), now.toLocalTime(), Year.now().getValue() + 1, "U".getBytes(), "P".getBytes(),
+                    "D".getBytes(), "A".getBytes(), "T".getBytes(), "E", "text", "mediumText", "3", "3", PipelineCaseHelper.generateJsonString(32, true), orderId});
+            return;
+        }
+        if (databaseType instanceof PostgreSQLDatabaseType || databaseType instanceof OpenGaussDatabaseType) {
+            String updateSql = String.format(POSTGRESQL_UPDATE_ORDER_BY_ID, orderTableName);
+            DataSourceExecuteUtil.execute(dataSource, updateSql, new Object[]{"中文测试", randomInt, BigDecimal.valueOf(10000), true, "bytea-update".getBytes(), "update", 1.0F, 2.0,
```

Comment:
Could we use real binary to test but not ascii chars. e.g. `new byte[] {-1, 0, 1}`